### ISSUE TITLE
Connect device modal is confusing

### ISSUE THREAD

**Issue by jamesadevine** [microsoft/jacdac#842]
![image](https://user-images.githubusercontent.com/4742697/142176088-4a4beb1a-3bd1-45bd-ba7e-543aa964520f.png)

I am confused by the connect device modal. I know the design intention is that you pick USB or serial depending on the brain you have.

It is unusual for a button to appear as the title of each section. My instinct was to click on the picture of the thing I have as opposed to identifying the brain I have and clicking on "connect USB" or "connect serial".

I think the header "connect USB"/"connect Serial" buttons should be removed in place of a connect button for each device picture. The user does not care between the distinction of USB/Serial, that is an implementation detail. The button should just be connect.

**Comment by pelikhan** [comment]
Clicking on images is misleading because we don’t choose specific hardware when connection,  we connect to *any* usb device or *any* serial device.
________________________________

**Comment by jamesadevine** [comment]
I was not proposing clicking on images! I was suggesting that each device has a connect button, but agree that's also misleading. 

It might be best to move the connect button below the rows of the devices. The current connect button does not follow the users attention. Users typically look top to bottom unless their attention is drawn in a particular direction. The current dialog directs user attention to the images and then the user has to look above the images to find the button they are supposed to click.

This is how my attention was drawn in the dialog:

![image](https://user-images.githubusercontent.com/4742697/142228797-e28c458b-e4e1-4db8-88ee-a4d52001dcf2.png)

I clicked on the micro:bit picture automatically because the interaction was not clear. I did not see the green button as my attention was not drawn to it by the dialogue design.
